### PR TITLE
Datacatalog : Added ITSystem Test

### PR DESCRIPTION
**Pull request by athakor**

### CODE REVIEW

File: google-cloud-clients/google-cloud-datacatalog/src/test/java/com/google/cloud/datacatalog/v1beta1/it/ITSystemTest.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+  @BeforeClass
+  public static void beforeClass() throws Exception {
+    client = DataCatalogClient.create();
+    client.createTagTemplate(PARENT, TAG_TEMPLATE_ID, TAGTEMPLATE);
+    client.createEntryGroup(PARENT, ENTRY_GROUP_ID, ENTRY_GROUP);
+  }
+
+  @AfterClass
+  public static void AfterClass() {
```
Comment (rahulKQL) [inline google-cloud-clients/google-cloud-datacatalog/src/test/java/com/google/cloud/datacatalog/v1beta1/it/ITSystemTest.java:99]: change the method name to `afterClass()`

**Review by pmakani — APPROVED** [review]
LGTM

File: google-cloud-clients/google-cloud-datacatalog/src/test/java/com/google/cloud/datacatalog/v1beta1/DataCatalogHelper.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ */
+package com.google.cloud.datacatalog.v1beta1;
+
+import java.util.UUID;
+
+public class DataCatalogHelper {
+
+  private static final String TAG_TEMPLATE_ID = "test_tag_template_id_";
+  private static final String TAG_TEMPLATE_FIELD_ID = "test_tag_template_field_id_";
+  private static final String ENTRY_GROUP_ID = "test_entry_group_id_";
```
Comment (rahulKQL) [inline google-cloud-clients/google-cloud-datacatalog/src/test/java/com/google/cloud/datacatalog/v1beta1/DataCatalogHelper.java:24]: I think this is not needed. It is being used only once on their respective generate mehtod you can directly use string.

File: google-cloud-clients/google-cloud-datacatalog/src/test/java/com/google/cloud/datacatalog/v1beta1/it/ITSystemTest.java (hunk 1 above)
Comment (athakor) [reply google-cloud-clients/google-cloud-datacatalog/src/test/java/com/google/cloud/datacatalog/v1beta1/it/ITSystemTest.java:99]: > change the method name to `afterClass()`

done

**Pull request closed without merging**